docs/sidebar.md

```diff
@@ -1,6 +1,6 @@
 - **User Guide**
 - [Introduction](introduction)
-- [Tiramisu](tiramisu/sd-preparation)
+- [Aroma](aroma/sd-preparation)

```

Comment:
shouldn't this link to the getting started page?

Reply:
yes it should

Reply:
fixed. ~~this is why we have PRs~~